imap/mailbox.c

```
@@ -5715,104 +5698,18 @@ static int _mailbox_index_repack(struct mailbox *mailbox,
-                        struct index_record newcrcrec = *record;
-                        newcrcrec.internaldate = frock.internaldate;
-                        repack->crcs.basic ^= crc_basic(mailbox, &newcrcrec);
-                    }
-                    else {
-                        // make sure we don't have a JMAP ID (internaldate) clash
-                        conversations_adjust_internaldate(cstate, guid,
-                                                          &copyrecord.internaldate);
-                    }
-                }
-
-                if (!dryrun) {
-                    // track this record so we can set the file timestamps later
-                    // we only need UID, internaldate, and internal_flags
-                    struct index_record *trecord =
-                        xzmalloc(sizeof(struct index_record));
-
-                    trecord->uid = copyrecord.uid;
-                    trecord->internaldate = copyrecord.internaldate;
-                    trecord->internal_flags = copyrecord.internal_flags;
-                    ptrarray_append(records, trecord);
-                }
-
-                // update G & J records
-                r = mailbox_update_conversations(mailbox, record, &copyrecord);
-                if (r) goto done;
-
-                if (mailbox->i.minor_version < 20) {
-                    // add virtual annotation to old crc
-                    buf_reset(&buf);
-                    buf_printf(&buf, UINT64_FMT, copyrecord.internaldate.tv_nsec);
-                    repack->crcs.annot ^=
-                        crc_annot(record->uid,
-                                  IMAP_ANNOT_NS "internaldate.nsec", "", &buf);
-                }
+                copyrecord.internaldate.tv_nsec = nsec;
             }
             buf_reset(&buf);
             r = annotate_state_writesilent(astate, IMAP_ANNOT_NS "internaldate.nsec", "", &buf);
-            if (r) goto done;
```

Why can we safely ignore this error?